templates/workflow/workflow.tpl

```diff
@@ -58,6 +58,14 @@
 			>
 				{translate key="common.preview"}
 			</pkp-button>
+			{if $submissionPaymentsEnabled}
```

Comment:
This is nice in that it doesn't have any impact on the UI if they're not using payments. It looks like you were able to use the UI library components really well! How did you find it? Easy enough or still an uncomfortable way of working?

Reply:
This was a really helpful exercise (I was long overdue to start working with this stuff meaningfully). I still could use some (re)orientation on Swagger/Redoc -- maybe that's a missing aspect of our tech documentation?

Thanks for the cleanup -- the `PUT`/`POST` stumbling block had me stumped but the error handling/authorization(!)/other patterns were useful guidance too.

Reply:
Frankly just some basics on what tool to use where. For example, are we using swagger-editor?

Reply:
Ah ok. The `swagger-source.json` file is an OpenAPI 3.0 spec, so we can use different tools. We no longer use swagger to generate our docs, but I often use the swagger-editor to quickly check the changes I've made. I'll make a note to write something up about documenting changes to the API.

Reply:
:+1: Thanks!